bootstrap/_deb_common.sh

```diff
@@ -44,6 +44,10 @@ apt-get install -y --no-install-recommends \
   libffi-dev \
   ca-certificates \
 
+if [[ $? > 0 ]] ; then
```

Comment:
Does this cause an exit in the case where one of the packages was simply not available on some Debian variant?  If it does, we will need to be extremely careful about the change, because we've seen incredible diversity in what some packages are called, and erroring out because of it will require a lot more careful logic to figure out the right names for virtualenv and python packages especially.